docs/configuration/config-file-reference.md

```diff
@@ -271,6 +271,15 @@ query_scheduler:
     # CLI flag: -query-scheduler.grpc-client-config.connect-timeout
     [connect_timeout: <duration> | default = 5s]
 
+resource_thresholds:
```

Comment:
IMHO, it's worth mentioning what metrics are being used to estimate resource usage, and it's probably worth mentioning that they might not exactly translate to memory usage, for example.

Reply:
Makes sense. I will clarify it with more comments